plugins/uefi-dbx/fu-uefi-dbx-plugin.c

```diff
@@ -38,7 +53,30 @@ fu_uefi_dbx_plugin_coldplug(FuPlugin *plugin, FuProgress *progress, GError **err
 		fu_device_inhibit(FU_DEVICE(device),
 				  "no-dbx",
 				  "System firmware cannot accept DBX updates");
+		inhibited = TRUE;
+	}
+
+#ifdef WITH_UEFI_DBX_SNAPD_NOTIFIER
+	/* TODO progress? */
+	if (FU_UEFI_DBX_PLUGIN(plugin)->snapd_notifier != NULL) {
```

Comment:
nah, just do a `self = FU_UEFI_DBX_PLUGIN(plugin)` at the top of the function pls.